#battery
string name
int16 id
string chemistry
float32 rated_voltage
float32 current_voltage
float32 rated_capacity_Ah
float32 current_capacity_Ah
float32 capacity_level_perc
uint8 cells_in_series
uint8 cells_in_parallel
bool active
bool recharging
cell[] cells